styles/components/_button.scss

```diff
@@ -7,18 +7,16 @@ button {
   color: #707070;
 }
 
-.nav-call {
```

Comment:
Pulled the buttons out of their parent classes and gave them unique names

This makes it so we can use these buttons anywhere, rather than having to copy and paste the code for each parent that a button lives in

Also, no styling elements directly 😉 Always use a class if you're going to style something!